### PR TITLE
Add inline documentation for `sequence`

### PR DESCRIPTION
**Pull request by gfontenot**

### CODE REVIEW

**Comment by edwardloveall** [comment]
Looks good to me

**Comment by tonyd256** [comment]
:+1:

File: Argo/Functions/sequence.swift [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/**
+  Convert an `Array` of `Decoded<T>` values to a `Decoded` `Array` of unwrapped
+  `T` values
```
Comment (tonyd256) [inline Argo/Functions/sequence.swift]: missing `.`

File: Argo/Functions/sequence.swift [hunk 2]
```diff
@@ -12,6 +26,22 @@ public func sequence<T>(xs: [Decoded<T>]) -> Decoded<[T]> {
   return pure(accum)
 }
 
+/**
+  Convert a `Dictionary` with `Decoded<T>` values to a `Decoded` `Dictionary`
+  with unwrapped `T` values
```
Comment (tonyd256) [inline Argo/Functions/sequence.swift]: missing `.`

**Pull request merged**